catalyst/dl/callbacks/metrics/accuracy.py

```diff
@@ -9,7 +9,8 @@
 
 
 class AccuracyCallback(BatchMetricCallback):
-    """Accuracy metric callback.
+    """
+    Accuracy metric callback.
```

Comment:
by default docstrings have the following structure:
```
"""Short inline description.

Long-long-long
multi-line description

Args (and other blocks like Raises, Yields, Returns, Examples):
    sub-items (with indent)

"""
```
Are we going to follow this structure or you proposing another one?